src/assets/manifest.json

```diff
@@ -38,5 +38,6 @@
       "type": "image\/png",
       "density": 4
     }
-  ]
+  ],
+  "gcm_sender_id": "000000000000"
```

Comment:
We need the id from the Google Developer Console (and activate the Firebase Cloud Messaging API first)